packages/core/src/performers/auto-performer/AutoPerformer.ts

```diff
@@ -221,11 +221,11 @@ export class AutoPerformer {
     );
 
     for (let step = 0; step < maxSteps; step++) {
-      const screenCapture = await this.screenCapturer.capture();
+      const screenCaptureWithoutHighlight = await this.screenCapturer.capture(false);
```

Comment:
it is not a `chore` PR if we're changing the behavior. I renamed your PR - see the change